docs/auto-release-workflow.md

```
@@ -0,0 +1,1199 @@
+
+- [Overview](#overview)
+- [Key Differences from Manual Process](#key-differences-from-manual-process)
+- [Prerequisites](#prerequisites)
+- [Release Workflow Stages](#release-workflow-stages)
+  - [1. Pre-Merge: Development and Configuration](#1-pre-merge-development-and-configuration)
+  - [2. Automated Pipeline Execution: Build and Version Synchronization](#2-automated-pipeline-execution-build-and-version-synchronization)
+  - [3. Auto Release Workflow](#3-auto-release-workflow)
+- [Configuration](#configuration)
+  - [ReleasePlan Configuration](#releaseplan-configuration)
+  - [ReleasePlanAdmission Configuration](#releaseplanadmission-configuration)
+  - [Service Account Setup](#service-account-setup)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)